packages/module/patternfly-docs/content/extensions/virtual-assistant/examples/demos/AttachmentDemos.md

```diff
@@ -74,17 +74,17 @@ This demo includes broader chatbot features, including:
 
 This demo displays unique attachment features, including:
 1. [`<ChatbotContent>` and `<MessageBox>`](/patternfly-ai/chatbot/chatbot#chatbot-content-and-message-box) components that contain:
-- A PatternFly [`<Alert>`](/components/alert) to provide success or error messages for attachments.
+    - A PatternFly [`<Alert>`](/components/alert) to provide success or error messages for attachments.
```

Comment:
I don't know if we want to specifically call out that this is a slightly custom PatternFly alert so it can sticky within the chatbot. If they use a regular PatternFly alert, it's not going to look like the demo.